wordpress-mobile/MediaPicker-iOS
# Fixing WPBadgeView Autolayout Warning

### Details
Fixes #298

### To test:
1. Checkout [commit 9788de0 (in this branch)](https://github.com/wordpress-mobile/MediaPicker-iOS/commit/9788de0f02c9bee484ec5b5d66aeb499f1fab5ee).
2. Launch the sample app
3. Press the top right `+` button > `Camera Roll`

Verify there's a badge on top of every image, and no error shows up in the console.

@SergioEstevao would you be game for a really fun PR?
Thanks in advance!!
